student.rb

```diff
@@ -1,12 +1,19 @@
+require_relative 'classroom'
+
 class Student < Person
   attr_accessor :classroom
 
-  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
+  def initialize(age, name = 'Unknown', parent_permission: true)
     super(name, age, parent_permission)
-    @classroom = classroom
+    @classroom = nil
```

Comment:
-I suggest initializing this classroom variable like this `@classroom=classroom`. Please add the `classroom` variable as your instance variable 🙏